src/java/com/zimbra/qa/soap/RestServletTest.java

```diff
@@ -360,7 +360,7 @@ private boolean containsText(String contentType) {
 	}
 
 	private boolean isBinary(String contentType) {
-		if (contentType.equalsIgnoreCase("application/zip")) {
+		if (contentType.equalsIgnoreCase("application/zip") || contentType.equalsIgnoreCase("application/x-compressed-tar")) {
```

Comment:
Does this affect legacy rest tests in any way?

Reply:
nope